### PR TITLE
docs(readme): proof reading [ci skip]

### PR DESCRIPTION
**Pull request by Sorunome**

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request, please fix them before in order to have your pull-request merged.

 * Commit: 7b4b23277f8327a0b7bc6f41cb6a8b5235cd35f1
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/RicinApp/Ricin/blob/master/CONTRIBUTING.md#-git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

File: ReadMe.md [hunk 1]
```diff
@@ -47,12 +47,12 @@ while rewriting Ricin.
 That's why I wrote this statement, donations are not always about money, you can also donate me some love, feedbacks,
 report the bugs you've seen, contribute to the code, pay me a bear (wuups, a beer, eheh).
 
-Of course if you really wish to give me some money for that work, i won't reject it. But before donating money, ask
-yourself "Do he really merits that money? Do I believe in privacy enough to give some cents to this crazy guy?". If you
+Of course if you really wish to give me some money for that work, I won't reject it. But before donating money, ask
+yourself "Dos he really merit that money? Do I believe in privacy enough to give some cents to this crazy guy?". If you
```
Comment (SkyzohKey) [inline ReadMe.md]: Wouldn't `Does he` be more appropriated here ?

Comment (Sorunome) [reply ReadMe.md]: Woops, yeah, typo, imma edit

**Comment by GitCop** [comment]
There were the following issues with your Pull Request, please fix them before in order to have your pull-request merged.

 * Commit: cf503272b9a6ed3f40c1d6c1419787cfc10425f0
  * Invalid type. Valid types are **fix, docs, style, refactor, perf, test, chore, revert, feature, feat, ui, i18n**

Guidelines are available at https://github.com/RicinApp/Ricin/blob/master/CONTRIBUTING.md#-git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Pull request merged**